source/material_model/visco_plastic.cc

```diff
@@ -263,6 +259,10 @@ namespace aspect
               }
             }
 
+          // Step 1c-1: If selected, replace the current pre-yield viscosity with the Frank Kamenetskii viscosity
+          if (viscous_type == frank_kamenetskii)
+            viscosity_pre_yield = frank_kamenetskii_rheology.compute_viscosity(temperature, j);
```

Comment:
Can you put this in line 250?

Reply:
Yep, I definitely did not need to add these lines.

Reply:
Updated with the last commit.